### DISCUSSION TITLE
Chapter on Cultural Differences when Approaching OSS

### DISCUSSION THREAD

**Discussion by dicortazar** [todogroup/ospology#511] [Community Proposals]
Hi everyone, 

**Context**

In the past years, I've been discussing with [Willem](https://github.com/willemjiang) on cultural differences when approaching OSS projects and how to engage developers from several places around the world effectively.

We had the opportunity to share our insights in [FOSSASIA](https://www.youtube.com/watch?v=RJzVTNXkaKE&list=PLzZVLecTsGpK2fr2MQQUSg1a1nWegsIRN) [Youtube video until minute 32] and lately at the [OSS Summit Europe](https://www.youtube.com/watch?v=J_GdZ7ziP4E&list=PLbzoR-pLrL6rC7SpO7MJCZm22Qp5ns3p-). In both cases we had good feedback and questions from the audience.

In addition, we had discussions on running workshops with some of the attendees and we're expanding a bit the scope by having a panel at the OSS Summit Japan as well in a few days.

**Why this matter to OSPOs?**

We have noticed that communities tend to be homogeneous by inertia, and this harms innovation. We have noticed at the same time that there are strategies we can all follow to mitigate the lack of geographical representation in communities and grow the awareness on this.

The accumulated knowledge, if we're able to express this in written form, would be valuable for everyone willing to engage people from any region around the world, and specially building bridges between western and eastern communities.

**Topics to cover**

The way we've structured the talk could be the topics to cover here:
* Cultural sensitivity in open source collaboration
* Tools to approach cultural differences
* More than just code
* Lessons to apply to OSS communities

**Next Step**

We've considered several options, from creating and starting a new place for discussion, to contributing this to some OSS projects. The TODO group may be this place, but we'd like to learn from you if this fits, if this would be useful, and some guidance on next steps.

Thoughts? :)

**Comment by anajsana** [comment]
Following up on the related panel we did today, Monday October 28th at OSSummit Japan.

The audience shared some keywords on what cultural differences meant to them via slido platform. 

- Here are the keywords

## Snapshot - Keyword Cloud

<img width="1255" alt="Screenshot 2024-10-28 at 17 38 47" src="https://github.com/user-attachments/assets/b100723f-ed37-44ec-b63d-4c2f56e083b5">


Also, we asked the audience what questions are interested in tackling, which can serve as a starting point for building content

- [Here](https://wall.sli.do/event/1T9sZKZiMLQLQKy39QBzfo?section=221df7a9-2487-45ca-82d1-963b0be17258) are the Questions raised


## Snapshot - Questions


1. How do you detect a potential problem based on cultural differences early on? Share your worst story. 

2. How do you hire people w/ cultural differences? I run a TSC with 15 ppl from APAC, EU, and the US. All white males. Any tips on how we can for more diversity?

3. What do we do with people who are not ammendable to working in a diverse environment and are always causing trouble?

4. How do you manage the challenge of developers from different cultures joining an OSS project that started in a specific country's culture?

**Reply by anajsana** [reply]
Here is the recording video from the panel discussion: https://youtu.be/Pk436X0HG5U?si=S4B2J2GhduuYEvIn

**Comment by annania** [comment]
Yes, I've had some conversations about this with some TODO Group folks. I think this is useful to acknowledge. Since this is a cultural topic, this will need to be done carefully to not make over generalizations and misrepresentations, to not offend any cultural group. I recommend seeking voices of folks from those communities to share their point of view.
